eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -8,6 +8,7 @@ parameters:
   PreSteps: []
   PostSteps: []
   TimeoutInMinutes: 60
+  TestMode: ''
```

Comment:
I don't know if there are any other direct usages of this template but I think it is probably worth defaulting this to Live as well.

Reply:
I'll circle back to this in a follow-up PR as not to trigger another ~300 pipelines